packages/roosterjs-editor-api/lib/format/toggleBlockQuote.ts

```diff
@@ -0,0 +1,119 @@
+import execFormatWithUndo from './execFormatWithUndo';
+import getFormatState  from './getFormatState';
+import getBlockQuoteElement from './getBlockQuoteState';
+import { ContentScope, ContentPosition, NodeBoundary } from 'roosterjs-editor-types';
+import { unwrap, wrapAll, wrap } from 'roosterjs-editor-dom';
+import { Editor, browserData } from 'roosterjs-editor-core';
+
+var ZERO_WIDTH_SPACE = '&#8203;';
+
+export default function toggleBlockQuote(editor: Editor, styler?: (element: HTMLElement) => void): void {
+    editor.focus();
+    let formatter: () => void = null;
+    let formatState = editor ? getFormatState(editor) : null;
+    let contentTraverser = editor.getContentTraverser(ContentScope.Selection);
+    let range = editor.getSelectionRange();
+
+    if (!formatState || !contentTraverser || !range) {
+        return;
+    }
+
+    let blockElement = contentTraverser.currentBlockElement;
+
+    if (!formatState.isBlockQuote) {
```

Comment:
You only need isBlockQuote here, why are you calling getFormatState() to get all format state?

Reply:
Fixed by calling isSelectionInBlockQuote()